data/random-teams.ts

```diff
@@ -2448,6 +2449,299 @@ export class RandomTeams {
 
 		return team;
 	}
+
+	randomBSSFactorySets: AnyObject = require("./bss-factory-sets.json");
+
+	randomBSSFactorySet(
+		species: Species,
+		teamData: RandomTeamsTypes.FactoryTeamDetails
+	): RandomTeamsTypes.RandomFactorySet | null {
+		const id = toID(species.name);
+		const setList = this.randomBSSFactorySets[id].sets;
+
+		const movesMax: { [k: string]: number } = {
+			batonpass: 1,
+			stealthrock: 1,
+			toxicspikes: 1,
+			trickroom: 1,
+			auroraveil: 1,
+		};
+		const weatherAbilities = [
+			"drizzle",
+			"drought",
+			"snowwarning",
+			"sandstream",
+		];
+		const terrainAbilities: { [k: string]: string } = {
+			electricsurge: "electric",
+			psychicsurge: "psychic",
+			grassysurge: "grassy",
+			seedsower: "grassy",
+			mistysurge: "misty",
+		};
+		const terrainItemsRequire: { [k: string]: string } = {
+			electricseed: "electric",
+			psychicseed: "psychic",
+			grassyseed: "grassy",
+			mistyseed: "misty",
+		};
+
+		const maxWantsTera = 2;
+
+		const requiredMoves: { [k: string]: number } = {};
```

Comment:
Since this is empty/unused, you should remove any code relating to `requiredMoves`, including `priorityPool`.

Reply:
Great, thanks! It's outside the scope of this PR but `requiredMoves` and `priorityPool` also exist in Gen 8 (and I think Gen 7?) BSS Factory code too and are likewise unused there, so this change should probably be made in that code too.